dojo/tools/nikto/parser.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, filename, test):
 
         tree = ET.parse(filename)
         root = tree.getroot()
-        scan = root.find('scandetails')
+        scan = root.find('./niktoscan/scandetails')
```

Comment:
Replacing the old with the new breaks older scans like [this one](https://github.com/DefectDojo/sample-scan-files/blob/master/nikto/nikto.xml). Maybe add this:
```
scan = root.find('scandetails') 
if scan is None:
    scan = root.find('./niktoscan/scandetails')
```

Reply:
Thank you @Maffooch , that is a fantastic proposal to maintain backwards compatibility. 

I have amended my commit based on your suggestion and have pushed it. After is done building, it should be ready for your approval,